samples/ChangeFeedManagement/Program.py

```diff
@@ -0,0 +1,125 @@
+import azure.cosmos.documents as documents
+import azure.cosmos.cosmos_client as cosmos_client
+import azure.cosmos.errors as errors
+import datetime
+
+import samples.Shared.config as cfg
+
+# ----------------------------------------------------------------------------------------------------------
+# Prerequistes - 
+# 
+# 1. An Azure Cosmos account - 
+#    https:#azure.microsoft.com/en-us/documentation/articles/documentdb-create-account/
+#
+# 2. Microsoft Azure Cosmos PyPi package - 
+#    https://pypi.python.org/pypi/azure-cosmos/
+# ----------------------------------------------------------------------------------------------------------
+# Sample - demonstrates how to consume the Change Feed and iterate on the results.
+# ----------------------------------------------------------------------------------------------------------
+
+HOST = cfg.settings['host']
+MASTER_KEY = cfg.settings['master_key']
+DATABASE_ID = cfg.settings['database_id']
+COLLECTION_ID = cfg.settings['collection_id']
+
+database_link = 'dbs/' + DATABASE_ID
+collection_link = database_link + '/colls/' + COLLECTION_ID
+
+class IDisposable(cosmos_client.CosmosClient):
+    """ A context manager to automatically close an object with a close method
+    in a with statement. """
+
+    def __init__(self, obj):
+        self.obj = obj
+
+    def __enter__(self):
+        return self.obj # bound to target
+
+    def __exit__(self, exception_type, exception_val, trace):
+        # extra cleanup in here
+        self = None
+
+class ChangeFeedManagement:
+    
+    @staticmethod
+    def CreateDocuments(client):
+        print('Creating Documents')
+
+        for i in range(1, 1000):
+            c = str(i)
+            document_definition = {'id': 'document'+ c,
+                                'address': {'street': '1 Microsoft Way'+c,
+                                            'city': 'Redmond'+c,
+                                            'state': 'WA',
+                                            'zip code': 98052
+                                            }
+                                }
+
+            created_document = client.CreateItem(
+                collection_link,
+                document_definition)
+
+    @staticmethod
+    def ReadFeed(client):
+        print('\nReading Change Feed from the beginning\n')
+
+        options = {}
+        # For a particular Partition Key Range we can use options['partitionKeyRangeId']
+        options["startFromBeginning"] = True
```

Comment:
It will be good to demonstrate options other than startFromBeginning too. Maybe insert few documents, do a readChangeFeed with 'startFromBeginning', get the time stamp, insert more documents , readChangeFeed with 'startTime' = TimeStamp

Reply:
Good idea, just added a sample for it